### PR TITLE
!!!TASK: Upgrade to latest neos/behat version

### PR DESCRIPTION
**Pull request by johannessteu**
Test distribution for travis:
https://travis-ci.com/johannessteu/neos-development-distribution/builds/97495679

### CODE REVIEW

**Comment by johannessteu** [comment]
Depends on https://github.com/neos/behat/pull/14

File: Neos.Neos/Tests/Behavior/Features/Bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -144,15 +140,15 @@ public function theFollowingUsersExist(TableNode $table)
...
+        $this->persistAll();
     }
 
     /**
      * @Given /^I am authenticated with "([^"]*)" and "([^"]*)" for the backend$/
      */
     public function iAmAuthenticatedWithAndForTheBackend($username, $password)
     {
-        $this->visit('/neos/login');
+        $this->visit('https://t3n.de/');
```
Comment (kdambekalns) [inline Neos.Neos/Tests/Behavior/Features/Bootstrap/FeatureContext.php]: Ad injection? ;) But honestly, does it work?

Comment (johannessteu) [reply Neos.Neos/Tests/Behavior/Features/Bootstrap/FeatureContext.php]: hahaha, lol, nope. The test's aren't executed at all cause they're tagged with @browser. It's a debug leftover where i wanted to test if selenium could handle absolute url's

**Comment by johannessteu** [comment]
All related PR's:
https://github.com/neos/behat/pull/14
https://github.com/neos/neos-development-collection/pull/2343
https://github.com/neos/flow-development-collection/pull/1496

**Pull request merged**